terraform/sentinel/README.md

```diff
@@ -0,0 +1,317 @@
+
+Copy the following files from this repository to your new repository:
+
+- `.github/workflows/terraform-plan.yml`
+- `.github/workflows/terraform-apply.yml`
+- `main.tf`
+- `terraform.tfvars.example`
+- `variables.tf`
+- `provider.tf`
+- `.gitignore`
+- `policies.sentinel`
+
+#### Step 3: Setup the GitHub repository secrets
+
+In order for the GitHub Actions to run, we need to set the following secrets in the GitHub repository:
+
+- `TF_API_TOKEN`: Terraform Cloud API token
+- `TF_CLOUD_ORGANIZATION`: Terraform Cloud organization name
+- `TF_WORKSPACE`: Terraform Cloud workspace name
+
+#### Step 4: Commit and push the changes
+
+Commit and push only the following files as the rest of the files will be added in the next step as part of a Pull Request:
+
+```bash
+git add .github/workflows .gitignore terraform.tfvars.example
+git commit -m 'Initial commit'
+git push origin main
+```
+
+#### Step 5: Create a Pull Request
+
+Create a branch and commit and push the rest of the files to the branch. Then create a Pull Request. Once the Pull Request is created, you should see the GitHub Actions running. Once the GitHub Actions have been completed, you should see a comment on the Pull Request with the results of the plan. If you would like to see what a failing plan looks like due to a policy check, you can change the `whois_privacy_enabled` to `false` in the `main.tf` file, or any of the other values in the Terraform configuration.
+
+```bash
+git checkout -b add-terraform-config
+git add main.tf variables.tf provider.tf
+```
+
+#### Step 6: Merge the Pull Request
+
+Once you have reviewed the changes and are happy with them, you can merge the Pull Request. Once the Pull Request is merged, you should see the GitHub Actions running again. This time, the GitHub Actions will run the `terraform apply` command and apply the changes to your DNSimple account. You can verify that the changes were applied by checking your DNSimple account and also inspecting the state and run logs in Terraform Cloud.
+
+## Conclusion
+
+Congratulations! You have successfully completed the "DNSimple Policy-as-Code with Terraform and Sentinel" demo. You have learned how to utilize Terraform, Sentinel, and DNSimple to establish a policy-driven DNS management framework, empowering your organization to democratize DNS management while ensuring policy adherence.
+
+When thinking about moving to production and maturing your policy-as-code framework, here are some things to consider:
+
+- **Modularize Policies**: In the current demo we created the policy manually in Terraform Cloud, however, to make the policy more reusable and easier to maintain, you can create a module for each policy and load them as a policy set in Terraform Cloud through version control. This will allow you to easily reuse the policies across multiple workspaces and organizations and also allow you to version control the policies with automated release processes and audit trails.
```

Comment:
```suggestion
- **Modularize Policies**: In the current demo, we created the policy manually in Terraform Cloud, however, to make the policy more reusable and easier to maintain, you can create a module for each policy and load them as a policy set in Terraform Cloud through version control. This will allow you to easily reuse the policies across multiple workspaces and organizations, and let you version control the policies with automated release processes and audit trails.
```